README.md

```diff
@@ -7,15 +7,198 @@
 
 ## Overview
 
-This reading is meant to give students a chance to catch their breath and review
-some of the big ideas that they've covered in recent lessons. Feel free to
-rehash/refactor previous examples — we want to make sure students have a strong
-base before plowing ahead too quickly.
+In this lesson we're going to revisit some of the concepts we have explored so
+far. Our focus will be on how components interact with each other and how global
+application state can be managed in a predictable, scalable way using actions,
+stores and handler function.
```

Comment:
'function' -> 'functions'

Also, what do you mean with handler functions? The component event handlers?